static/js/zamboni/devhub.js

```diff
@@ -124,6 +124,12 @@ $(document).ready(function() {
         $('.addon-upload-dependant').attr('disabled', !($(this).is(':checked')));
     });
 
+    $('#id_beta').bind('change', function() {
+        // TODO: Reset the form.
+        // I couldn't figure out how to do this easily. I was looking for a
+        // pre-existing function b/c I've seen this done elsewhere.
+    });
```

Comment:
We don't want to reset the form here, we want the message to be hidden and the submit button to be enabled.